README.md

```diff
@@ -62,6 +62,18 @@ Building `ghc-lib` is subject to the same minimum version requirements that appl
 | ghc-8.10.* | 8.6.5      |
 | ghc-8.11.* | 8.8.1      |
 
+### How do I use the `ghc-lib`/`ghc-lib-parser` version macros?
+
+In general we can interpret the phrase,
```

Comment:
I don’t want to duplicate the docs for this. https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/phases.html#standard-cpp-macros already describes the semantics. I would suggest a link + the sentence that HEAD is `0.x`.